tests/spread/extensions/snaps/env-injector-hello/snap/snapcraft.yaml

```diff
@@ -0,0 +1,18 @@
+name: env-injector-hello
+version: "1.0"
+summary: test the env-injector extension
+description: This is a basic snap for testing env-injector extension
+
+grade: devel
+base: core24
+confinement: strict
+
+apps:
+  env-hello:
+    command: usr/local/bin/hello
+    extensions: [ env-injector ]
```

Comment:
Add another app that set's its own `env_alias` and then test that we aren't overriding it.